native/linux-arm-clang/build.cake

```diff
@@ -0,0 +1,58 @@
+DirectoryPath ROOT_PATH = MakeAbsolute(Directory("../.."));
```

Comment:
This script is not complete nor does it work.

Seems libjpeg-turbo is having some issues:
 - https://github.com/libjpeg-turbo/libjpeg-turbo/issues/440
 - https://groups.google.com/g/skia-discuss/c/ZG784t39JyU
 - https://groups.google.com/g/skia-discuss/c/SGA9UVxjabc